client/src/thunks.ts

```diff
@@ -1,3 +1,4 @@
+import * as _ from "lodash";
```

Comment:
i think using lodash throws types away and replaces with `any`. if that's the case i'm not really a fan of using this library.

Reply:
I tried changing one of the types and it complained (both VSCode and the TS compiler), so I think we're okay.